servers/Azure.Mcp.Server/src/Azure.Mcp.Server.csproj

```diff
@@ -41,9 +41,39 @@
     <SelfContained>true</SelfContained>
     <IlcOptimizationPreference>Size</IlcOptimizationPreference>
     <IlcFoldIdenticalMethodBodies>true</IlcFoldIdenticalMethodBodies>
+    <CopyOutputSymbolsToPublishDirectory>false</CopyOutputSymbolsToPublishDirectory>
+  </PropertyGroup>
+
+  <!-- .NET Tool Package settings -->
+  <PropertyGroup Condition="'$(BuildNative)' == 'true'">
```

Comment:
I see the existing `/p:BuildNative` property is being leveraged here as well, does this mean these changes will enable the NuGet `McpServer` packaging to distribute the azmcp AOT binaries?

Reply:
thanks for the detailed response!